lib/puppet/functions/ensure_packages.rb

```diff
@@ -1,14 +1,13 @@
 # frozen_string_literal: true
 
-# THIS FILE WAS GENERATED BY `rake regenerate_unamespaced_shims`
-
 # @summary DEPRECATED.  Use the namespaced function [`stdlib::ensure_packages`](#stdlibensure_packages) instead.
```

Comment:
Hum, maybe also change this line because it is what triggers the rake task that generated this code:
https://github.com/puppetlabs/puppetlabs-stdlib/blob/main/Rakefile#L95

```suggestion
# @summary DEPRECATED.  Use the scoped and namespaced function [`stdlib::ensure_packages`](#stdlibensure_packages) instead.
```

Very hackish :-/

Otherwise we can search for `Puppet::Functions\.create_function\(.*, Puppet::Functions::InternalFunction\)` and generate different code if we find it.  Also a bit hackish.

Reply:
It's more an internal detail.  Not sure we really need to spell it out to users.  Just not break things for them! :)

Are we likely to be running the rake task anymore?

Reply:
@smortex Now that I've updated the rake task to ignore this file, I've just got what you were getting at!